### ISSUE TITLE
Home button fails to resume game (Regression)

### ISSUE THREAD

**Issue by orbea** [libretro/RetroArch#5127]
## Description

While playing a game using a gamepad the `home` button opens the menu, pressing it a second time should resume the game, except it is now unresponsive. The `F1` key still works as do custom hotkeys (`L1+R1+Sel+Start`). This only occurs when the `All Users Control Menu` is enabled. This occurs with more than one gamepad.

### Expected behavior

Pressing the `home` button should resume the game.

### Actual behavior

Pressing the `home` button brings up the menu, but fails to resume the game.

### Steps to reproduce the bug

1. Start RetroArch.
2. Enable `All Users Control Menu` in the input settings.
3. Start a game and press the `home` button to call the menu.
4. Press the `home` button again, it should fail to resume the game.

### Bisect Results

```
42e56c84321350990afe3d825677b8bdfc2e29fa is the first bad commit
commit 42e56c84321350990afe3d825677b8bdfc2e29fa
Author: Brad Parker <cbparker@gmail.com>
Date:   Sun Jun 11 12:28:39 2017 -0400

    fix all_users_control_menu logic

:040000 040000 dff1347792b19922d4c5668b5f038934389df417 b3f121abf13be0003e3c8a88ec5bdabc351c4249 M	input
```
42e56c84321350990afe3d825677b8bdfc2e29fa

### Version/Commit
You can find this information under Information/System Information

- `RetroArch-1.6.1-x86_64-1_SBo`

### Environment information

- OS: `Slackware64-current`
- Compiler:  `gcc-7.1.0`

**Comment by orbea** [comment]
@bparker06 Can you take a look at this?

**Comment by bparker06** [comment]
@orbea Is this home key a button on your gamepad that's been bound to the menu toggle hotkey?

**Comment by orbea** [comment]
Yes, here is the auto config file for one of the gamepads I tested.
https://github.com/libretro/retroarch-joypad-autoconfig/blob/master/udev/HORI_HORIPAD_ONE.cfg

**Comment by JuanVCS** [comment]
Just tried with a wired Xbox 360 controller under linux, udev driver. Same problem.

**Comment by orbea** [comment]
Another workaround is enable `Unified Menu Controls`. Then the `home` button works correctly when `All Users Control Menu` is also enabled. 

@bparker, can you reproduce it if `Unified Menu Controls` is off?

**Comment by rz5** [comment]
I am on Windows 10, using a Logitech F310 in Xinput mode and on latest RA version as provided by the buildbot, I cannot reproduce the bug.

**Comment by tuirennder** [comment]
Issue still exist for me on today's nightly (15 Aug). Windows 10 with xbox one controller.

**Comment by Laynee** [comment]
The bug is still there. I've run some tests on Windows 10 + RA 1.8.1 with the following controllers:

- Sony DualShock 4 (Bluetooth)
- Sony DualShock 4 (USB)
- Microsoft Xbox 360 Wired (USB)
- Hori Real Arcade Pro 3 Fight Stick for PlayStation 3 (USB)
- Thrustmaster Firestorm Dual Analog 3 for PC & Mac (USB)
- Mayflash SNES Controller Adapter for PC (USB)

Can't return to the game with any of them if `All Users Control Menu` is enabled.

**Comment by koladonia** [comment]
The issue still persists more than a year after last mention.
Tested using Xinput with DS3.

**Comment by zoltanvb** [comment]
I can't reproduce this (linux/udev) nowadays. It is probably solved.
